controller/src/beerocks/master/wfa_ca.cpp

```diff
@@ -166,6 +175,60 @@ bool wfa_ca::parse_params(const std::vector<std::string> &command_tokens,
             return false;
         }
 
+        if ((*std::next(it))[0] == '{') {
+            int len_command_str     = (*std::next(it)).length();
+            bool prev_space         = false;
+            bool prev_curly_bracket = false;
+            int count_open_brackets = 0;
+            std::string command_str = *std::next(it);
+            for (int i = 0; i < len_command_str; i++) {
+                auto c = command_str[i];
+                LOG(ERROR) << "char: " << c;
+                if (c == '{') {
+                    prev_curly_bracket = true;
+                    prev_space         = false;
+                    count_open_brackets++;
+                } else {
+
+                    if (count_open_brackets == 0) {
+                        err_string =
+                            "The following string is not valid (curly brackets test failed) : '" +
```

Comment:
I would change it to something like "Unbalanced curly brackets in value of param '<param_name>'"
No need to mention that the string is invalid because the return code is "INVALID".

Reply:
Ok, I'll change this .